neuraxle/steps/numpy.py

```diff
@@ -74,6 +76,26 @@ def __init__(self):
         # The concatenate is on the inner features so axis = -1.
         NumpyConcatenateOnCustomAxis.__init__(self, axis=-1)
 
+    def join_data_containers(
```

Comment:
Two comments on this: 
1. This method should be defined in NumpyConcatenateOnCustomAxis class
2. This method should not exist. Implement NumpyConcatenateOnCustomAxis.handle_* methods instead. This is probably why tests break.